public/css/list/comment-list.less

```diff
@@ -47,3 +47,9 @@
     white-space: normal;
   }
 }
+
+.controls .comment-list .list-item {
+  .visual {
+    width: auto;
```

Comment:
Can't we make this the default for minimal list items now? It's now added in three places, comment, host and service list items. The width of the minimal list items already differs in lists now (compared to common and detailed) and I don't see a reason why we should it make the default. The narrower visual on all minimal list items looks okay to me.

Reply:
Fine with that. See 03a2315f / f88dc369

✅